lib/utils/location.js

```diff
@@ -45,7 +45,7 @@ export function matchesPageLocation(includes, excludes) {
 /* eslint-disable key-spacing */
 export const regexes = {
 	frontpage:        /^\/(?:hot|new|rising|controversial|top)?(?:\/|$)/i,
-	comments:         /^\/(?:r\/([\w\.]+)\/)?comments(?:\/|$)/i,
+	comments:         /^\/(?:r\/([\w\.]+)\/)?comments(?:\/([a-z0-9]+)\/\w*)?(?:\/|$)/i,
```

Comment:
Also n.b. [we have tests for these](https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/254de3e52b177cb92f57a367954c0eb4af2a547d/lib/utils/__tests__/location.js#L46) which you'll have to change, at the very least to add the comment id capture group